engine/sound/src/sound.cpp

```diff
@@ -477,6 +492,26 @@ namespace dmSound
         return result;
     }
 
+    uint32_t GetMixRate()
+    {
+        return g_SoundSystem->m_MixRate;
+    }
+
+    uint32_t GetDefaultFrameCount(uint32_t rate)
+    {
+        if (rate == 48000)
+            return 1024;
```

Comment:
Empirically chosen for the unit tests to sound good.
(Also larger than 48000/60=800)

Not sure how to best calculate this number in a more robust fashion.